kalite/i18n/__init__.py

```diff
@@ -371,7 +371,7 @@ def update_jsi18n_file(code="en"):
     request.path = output_file
     request.session = {settings.LANGUAGE_COOKIE_NAME: code}
 
-    response = javascript_catalog(request, packages=('ka-lite.locale',))
+    response = javascript_catalog(request, packages=('ka-lite.locale',), domain="django")
```

Comment:
So if I'm understanding this correctly, some of the internationalized strings in JS files are not translated, but since the default domain `djangojs` is almost a superset of the `django` domain, your suggestion is that we should just use the `django` domain so that as much as possible is translated?